package.json

```diff
@@ -46,9 +46,10 @@
     "react": "~15.4.0",
     "react-bootstrap": "0.30.1",
     "react-dom": "~15.4.0",
-    "webpack": "4.19.1",
+    "react-p5-wrapper": "0.0.4",
```

Comment:
I needed to use this very old version of this package because any other versions needed Babel 7.

I can also do this without the package if we think this'll be an issue.